src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Query.cs

```diff
@@ -303,6 +306,12 @@ private int QueryFindReferencesStatus(OLECMD[] prgCmds)
             return VSConstants.S_OK;
         }
 
+        private int QuerySyncClassView(OLECMD[] prgCmds)
+        {
+            prgCmds[0].cmdf = (uint)(OLECMDF.OLECMDF_ENABLED | OLECMDF.OLECMDF_SUPPORTED);
```

Comment:
I `IOleCommandTarget` terms you can have menu items that are supported, but not enabled (faded out menu items).  This just says that we always support it, and it's always enabled.